### PR TITLE
Added files in debian subdirectory.

### PR DESCRIPTION
**Pull request by fike**
These files are base to build postgrest Debian package. Additional, it has TODO-deps.md file with whole task to build postgrest as Debian official package.

### CODE REVIEW

File: debian/copyright [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Files: *
+Copyright: Joe Nelson <joe@begriffs.com>
+License: Expat
+
+Files: debian/*
+Copyright: Fernando Ike <fike@midstorm.org>
+License: Expat
+
+License: Expat
+ Copyright (c) 2014 Joe Nelson
```
Comment (begriffs) [inline debian/copyright]: Maybe 2015 or remove the year?

**Comment by begriffs** [comment]
Thank you! Can you explain what this does in Debian? Will we need more documentation about how to use these files?

File: debian/copyright (hunk 1 above)
Comment (fike) [inline debian/copyright]: Certainly, I made a mistake. The better will be 2014-2015. :+1: 

I will fix.

**Comment by diogob** [comment]
Can't wait to apt-get install postgrest <3

iu-3

**Comment by begriffs** [comment]
Oh this allows us to apt-get install! I see now, so cool!

**Comment by begriffs** [comment]
There is also this Debian work in progress by @n3xtchen https://github.com/n3xtchen/postgrest/commit/153863efad5e3e1350cc38bed8b3207f29d72caf

Maybe that change will be necessary as well for a working Debian package? I'll have to read the [guide](https://wiki.debian.org/HowToPackageForDebian) to understand how this process works.

**Comment by fike** [comment]
I think so change a little bit the postgrest. However, still some library dependencies need to (debian) package. I created a TODO list file ([debian/TODO-deps.md](https://github.com/fike/postgrest/blob/master/debian/TODO-deps.md)) with the tasks need to build correctly.

I think the main problem to build the Postgrest is the library dependencies. If someone will want to help, the link @begriffs posted is first the all. Other good page to read is [Haskell CollabMaint Getting Started](https://wiki.debian.org/Haskell/CollabMaint/GettingStarted). 

Does @begriffs  think possible to merge this PR and of @n3xtchen?

**Pull request merged**